docs/src/gatsby-theme-docz/index.tsx

```diff
@@ -34,6 +34,10 @@ interface Props {
 const docLayout = css`
   display: flex;
   flex-direction: row;
+  .docContainer {
+    word-wrap: break-all;
```

Comment:
I think you mean `word-break: break-all;`, but that causes issues, with line-wraps mid-word.